Plugin.php

```diff
@@ -140,13 +140,25 @@ protected function configureProviders(): void
      */
     protected function extendAuthController(): void
     {
+        // Extend the auth controller to add the SSO login buttons only if the native auth is disabled
+        if(Config::get('winter.sso::prevent_native_auth'))  {
+            Event::listen('backend.page.beforeDisplay', function ($controller, $action) {
+                if(!$controller instanceof \Backend\Classes\Controller) {
+                    return;
+                }
+                if ($action == 'signin') {
+                    $controller->addViewPath(plugins_path('winter/sso/views/auth'));
+                }
+            });
+        }
```

Comment:
```suggestion
        // Extend the auth controller to add the SSO login buttons only if the native auth is disabled
        if (Config::get('winter.sso::prevent_native_auth', false))  {
            \Backend\Controllers\Auth::extend(function ($controller) {
                $controller->addViewPath(plugins_path('winter/sso/controllers/auth/prevent_native'));
            });
        }
```

Could we use the above method and move the view file to the above path to make it a bit more clear what is happening?

Reply:
Sure, I'll do that, thanks

Reply:
Yes i tested it its working well =)